lib/pf/pfcmd/report.pm

```diff
@@ -124,15 +124,16 @@ sub report_db_prepare {
         last_arp,
         last_dhcp,
         device_type AS os
-      FROM node
-      WHERE tenant_id = ?
-        AND mac NOT IN (
-          SELECT
-            mac
-          FROM ip4log
-          WHERE end_time = 0
-            OR end_time > NOW()
-        )
+      FROM node n LEFT JOIN ip4log i USING (tenant_id, mac)
```

Comment:
I don't think this file has a use anymore so we should remove it if that's the case